client/post-editor/media-modal/markup.js

```diff
@@ -148,13 +148,19 @@ Markup = {
 				urlOptions = { size: options.size };
 			}
 
-			const img = React.createElement( 'img', {
+			const isTransient = ( 0 === ( media.URL || '' ).indexOf( 'blob:' ) ) || !! media.transient;
```

Comment:
went ahead and modified the check only for `media.transient` property dc4415698f6c627f3cebdda5fe95f7d4bde9de5a